docs/general/ad-filtering/create-own-filters.md

```diff
@@ -250,20 +250,33 @@ Example:
 
 ### Basic modifiers {#basic-rules-basic-modifiers}
 
+The following modifiers are the most simple and frequently used.
+Basically, they just limit the scope of rule application.
+
 <!-- Please keep them sorted -->
 
-- [`$app`](#app-modifier)
-- [`$denyallow`](#denyallow-modifier)
-- [`$domain`](#domain-modifier)
-- [`$header`](#header-modifier)
-- [`$important`](#important-modifier)
-- [`$match-case`](#match-case-modifier)
-- [`$method`](#method-modifier)
-- [`$popup`](#popup-modifier)
-- [`$third-party`](#third-party-modifier)
-- [`$to`](#to-modifier)
+| Modifier \ Products | [CL apps][cl-apps] | [ExtChr][ext-chr] | [ExtFf][ext-ff] | [iOS app][ios-app] | [Safari][ext-saf] | [ExtEdg][ext-edg] | [CB][and-cb] |
```

Comment:
* `$match-case` is supported by `CL apps` and `Content Blocker`. NO support in AdGaurd for Safari/iOS (maybe a bug, not sure)
* `$denyallow` - checked, it does not work in `Content Blocker`
* `$important` - checked, it does not work in `Content Blocker`
* `$to` - if it will be implemented, it may be mentioned under the table.

Also I suggest to use "✅[1]"+comment below instead of `⚠️` in case of partial support